internal/rest/resources/control.go

```diff
@@ -59,6 +59,19 @@ func controlPost(state state.State, r *http.Request) response.Response {
 		return response.SmartError(err)
 	}
 
+	daemonConfig := trust.Location{Address: req.Address, Name: req.Name}
+	err = intState.SetConfig(daemonConfig)
```

Comment:
nit: error check can be inlined (here and below)

Reply:
We tend to avoid inline errors for consistency, but thanks for the suggestion.